src/main/kotlin/org/kamiblue/client/gui/hudgui/elements/combat/Armor.kt

```diff
@@ -140,11 +141,11 @@ object Armor : HudElement(
 
         if (itemStack.isItemDamaged && colorBarMode) {
             val health: Double = itemStack.item.getDurabilityForDisplay(itemStack)
-            val rgbForDisplay: Int = itemStack.item.getRGBDurabilityForDisplay(itemStack)
+            val hexRgb: Int = itemStack.item.getRGBDurabilityForDisplay(itemStack)
             val i = (13.0f - health.toFloat() * 13.0f).roundToInt()
```

Comment:
Rename this to length, but the magical calculation is weird here

Reply:
This is basically copied and pasted from mojangs code, what do you expect?

Reply:
I would expect a clean one instead of pasted directly from mojang code

Reply:
If we are going for similarity to the minecraft items then we can't implement a new one, unless I am misunderstanding you?